lesson12/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css"  />
+    <script src="./main.js" defer></script>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <input id="js-button" type="button" value="Click!" onclick="onButtonClick()" />
```

Comment:
Q.
Is there a reason you are using onclick property instead of addEventListener?
ESLint on CodeSandbox shows an error.

I was aware that onclick was deprecated because registering two events would cause overwriting.
I also thought it was unclear which elements were clicked on the script file.

Reply:
thank you for your comment Haru-san,
I was not sure the difference between using onclick and addEventListener() methods, as well as what you meant by "uncler". [This article](https://qiita.com/KKKarin/items/ccb8ed361ab9acd1f9cf) briefly explains about your points, the differences, and pros/cons of them. 
I replaced onclik with addEventListner() method. 
Kindly check my latest commit

Reply:
Thank you for your fixed!

>  "unclear which elements were clicked on the script file".

Sorry it is not clear.
On the JS file, the onButtonClick() function is only defined,
and I have to look at the HTML to see when and where the function is triggered.
I think that writing addEventListener in a JS file also has the advantage of making the code flow easier to understand.

Reply:
knowing other people's point of view is helpful me to understand what is a readable and traceable code.
Thank you Haru-san